Exporter/src/main/java/uk/ac/ebi/biostd/exporter/service/SubmissionService.java

```diff
@@ -57,52 +60,60 @@ public Submission getSubmission(String accNo) {
         return submission;
     }
 
-    private List<String> getAccessTags(Submission submission) {
+    private List<String> getAccessTags(Submission submission, List<String> accessTags) {
         List<String> tags = new ArrayList<>();
         tags.add(submissionDao.getUserEmail(submission.getOwner_id()));
-        tags.addAll(submissionDao.getAccessTags(submission.getId()));
+        tags.addAll(accessTags);
         tags.add("#" + submission.getOwner_id());
-
         return tags;
     }
 
-    private List<Attribute> getAttributes(Submission submission) {
+    private List<Attribute> getAttributes(Submission submission, List<String> accessTags) {
         List<Attribute> subAttributes = submissionDao.getAttributes(submission.getId());
-        subAttributes
-                .add(new Attribute("ReleaseDate", DateUtils.getFromEpochSeconds(Long.valueOf(submission.getRTime()))));
+        subAttributes.add(new Attribute("ReleaseDate", getFromEpochSeconds(parseLong(submission.getRTime()))));
         subAttributes.add(new Attribute("Title", submission.getTitle()));
+        accessTags.forEach(tag -> addProjectAttribute(subAttributes, tag));
         return subAttributes;
     }
 
+    private void addProjectAttribute(List<Attribute> subAttributes, String accessTag) {
+        if (accessTag.equals("Public")) {
+            return;
+        }
+
+        if (subAttributes.stream().noneMatch(attribute -> attribute.getName().equals(accessTag))) {
+            subAttributes.add(newAttribute("AttachTo", accessTag));
+        }
+    }
+
+    private Attribute newAttribute(String name, String value) {
```

Comment:
Since`Attribute` class already has a name/value constructor, I don't think we need this method